Doc/library/ssl.rst

```diff
@@ -2257,7 +2262,7 @@ SSL versions 2 and 3 are considered insecure and are therefore dangerous to
 use.  If you want maximum compatibility between clients and servers, it is
 recommended to use :const:`PROTOCOL_TLS_CLIENT` or
 :const:`PROTOCOL_TLS_SERVER` as the protocol version. SSLv2 and SSLv3 are
-disabled by default.
+disabled by default. ::
```

Comment:
I know I suggested this style in my earlier comment, but could you move :: to a new line so we don't lose `git blame` information? This also applies to the changes in Doc/library/pathlib.rst.